### ISSUE TITLE
Invariant/fuzz testing suite

### ISSUE THREAD

**Issue by nican0r** [woonetwork/WooPoolV2#111]
Hi, I took part in the audit competition on Sherlock and created an invariant/fuzz testing suite. I had tried to DM over discord but wasn't able to so am opening this issue as @fb-alexcq had suggested in the public chat. 

I'm happy to create a PR for the test suite but just wanted to make sure this was the right repo. 

I'm also attaching a brief report I wrote up about the tests I ran and some suggestions for expanding them
[Report_For_WooFi_Swap_FuzzingInvariant_Testing.pdf](https://github.com/woonetwork/WooPoolV2/files/14774297/Report_For_WooFi_Swap_FuzzingInvariant_Testing.pdf).

**Comment by fb-alexcq** [comment]
Yes, this is the right repo. @fb-alexcq  Do you mind adding me on Discord?  I could think about a bounty to you.

**Comment by fb-alexcq** [comment]
@nican0r Hi, you can push your fuzz testing here; also I opened up friend request on Discord.
